hack/observability/opentelemetry/controller-manager-patch.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       containers:
         - name: manager
           args:
-            - "--metrics-bind-addr=127.0.0.1:8080"
+            - "--metrics-bind-addr=:8080"
```

Comment:
@mboersma @sbueringer this fixes Prometheus metrics in tilt

Reply:
I think the problem is that the `ServiceMonitor` still points to the https (8443) port.

One solution would be to adjust the auth_proxy_service.yaml (8443=>8080, scheme:https => scheme:http, port https => metrics (and also add the corresponding metrics port (8080) to the deployment)